templates/master/01-master-kubelet/_base/units/kubelet-cleanup.service.yaml

```diff
@@ -0,0 +1,16 @@
+name: kubelet-cleanup.service
+enabled: true
+contents: |
+  [Unit]
+  Description=Kubernetes Kubelet After Reboot Cleanup
+  Before=kubelet.service
+
+  [Service]
+  Type=oneshot
+  ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+  ExecStartPre=/bin/rm -f /var/lib/kubelet/memory_manager_state
```

Comment:
I'd probably do ```
  ExecStart=/bin/rm -f /var/lib/kubelet/cpu_manager_state /var/lib/kubelet/memory_manager_state
```

In general just use `ExecStart`. If we can have multiple instances (need to check) turn `ExecStartPre` into `ExecStart`.

Reply:
I need to confirm that, but apparently we can hae multiple execs for oneshot services.